test/e2e/third-parties/index.test.js

```diff
@@ -0,0 +1,32 @@
+import { createNextDescribe } from 'e2e-utils'
```

Comment:
can we change this test file to typescript like others to avoid potential mistakes in test files